src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -3,6 +3,7 @@
 
```

Comment:
@vexx32, your last commit had 1 failures in `PowerShell-CI-windows`
Test-Connection.Ping.Repeat works
```powershell
Expected 32, but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Test-Connection.Tests.ps1: line 207
207:                 $pingResults.Where{ $_.Status -eq 'Success', 'Default', 1 }[0].BufferSize | Should -Be 32
```